packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -11128,8 +11120,6 @@ Generated by [AVA](https://ava.li).
       1. onPreBuild command from ./plugin                           ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
-    Netlify configuration property "functions.test" value changed to {}.␊
-    Netlify configuration property "functions.test.included_files" value changed to [].␊
     Netlify configuration property "functions.test" value changed to { included_files: [] }.␊
```

Comment:
Those deletions are expected, and good. Now, redundant changes are not logged anymore.